src/components/BoxVaultAPY/BoxVaultAPY.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { MouseEvent, ReactElement } from "react";
+import Token from "@yearn/yearn-web-assets";
+import styled, { css } from "styled-components";
+import { Heading1, Text } from "../../typography";
+import { Icon } from "..";
+import { IconDirection } from "../Icon/Direction";
+
+export interface BoxVaultAPYProps {
+  address: string;
+  chainID?: "1" | "250" | "42161";
+  imgFormat?: "img" | "svg";
+  label: string;
+  value: string;
+  onClick?: (event: MouseEvent<HTMLButtonElement>) => void;
+}
+
+export const BoxVaultAPY = ({
+  address,
+  chainID,
+  label,
+  value,
+  imgFormat,
+  ...props
+}: BoxVaultAPYProps): ReactElement => (
+  <StyledBox {...props}>
```

Comment:
I would add here `if (!address || !chainID || !value) return null;`

Reply:
Hum, the token lib has a fallback to an empty icon if the address/chainID does not match, and I would rather have no null here (could break layout) but this fallback to be displayed instead. 
And having no value but just the icon + title could be a thing too, but this would require to not change layout if the value is not here

Reply:
"Fixed" by 1f806eb

Reply:
Sounds sensible! 🙇